### PR TITLE
Fill in default notif modal elements for each class

### PR DESCRIPTION
**Pull request by eliasfang**
### Administrative Info
Monday Board ID: 853648344
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Removed unnecessary fields from the notif modal
- Filled in class name, instructor name, and date/time of class
- Changed buttons at bottom to be of type "reset" and "submit" for the backend
    - Clicking Cancel clears any changes
    - Clicking Submit will submit the form (saving not yet implemented)

### Testing
How did you confirm your changes worked? 
- Checked to make sure contents in table match contents in notif modal
- Checked to see that client modal popups up when "Show email list" is clicked"
- Checked that fields are editable

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

| First Class in Table | Notif Modal of First Class in Table | Popup client modal |
| --- | --- | --- |
| Screenshot from 2021-05-10 19-10-16 | Screenshot from 2021-05-10 19-10-40 | Screenshot from 2021-05-10 19-11-07 |

### CODE REVIEW

**Review by mingkaili — CHANGES_REQUESTED** [review]
Looks like the branch is a little out of date? the svgs, and some of Shravan's functions are deleted

File: app/views/components/modals/notifModal.ejs [hunk 1]
```diff
@@ -9,91 +9,32 @@
         <form class="col s12">
             <div class="row">
                 <div class="input-field col s12">
-                    <input type="text" placeholder="Class Name">
+                    <input type="text" placeholder="Class Name" value="<%=job.class_name%>">
```
Comment (mingkaili) [inline app/views/components/modals/notifModal.ejs]: As I mentioned in the meeting you need a name prop for all input ex: `name='class_name'`

File: app/views/components/modals/notifModal.ejs [hunk 2]
```diff
@@ -9,91 +9,32 @@
...
                 </div>
             </div>
             <div class="row">
                 <div class="input-field col s12">
-                    <input type="text" placeholder="Instructor">
+                    <input type="text" placeholder="Instructor" value="<%=job.instructor_first_name%> <%=job.instructor_last_name%>">
```
Comment (mingkaili) [inline app/views/components/modals/notifModal.ejs]: Same here

**Review by mingkaili — CHANGES_REQUESTED** [review]
Good job

File: app/views/components/modals/notifModal.ejs (hunk 1 above)
Comment (eliasfang) [reply app/views/components/modals/notifModal.ejs]: Added

File: app/views/components/modals/notifModal.ejs (hunk 2 above)
Comment (eliasfang) [reply app/views/components/modals/notifModal.ejs]: Added

File: app/views/components/modals/notifModal.ejs [hunk 3]
```diff
@@ -9,91 +9,32 @@
...
             <div class="row">
                 <div class="input-field col s12">
-                    <input type="text" placeholder="Class Name">
+                    <input type="text" name="class_name" placeholder="Class Name" value="<%=job.class_name%>">
                 </div>
             </div>
             <div class="row">
                 <div class="input-field col s12">
-                    <input type="text" placeholder="Instructor">
+                    <input type="text" name="instructor_name" placeholder="Instructor" value="<%=job.instructor_first_name%> <%=job.instructor_last_name%>">
```
Comment (mingkaili) [inline app/views/components/modals/notifModal.ejs]: This right here will cause trouble, when you submit, you will get one big string key in the backend, I suggest separate the first name and last name into 2 different fields, designers' didn't know that.... which is fine.

**Review by mingkaili — CHANGES_REQUESTED** [review]

File: app/views/components/modals/notifModal.ejs (hunk 3 above)
Comment (eliasfang) [reply app/views/components/modals/notifModal.ejs]: Separated

**Review by mingkaili — APPROVED** [review]
AMAZING!!

**Review by jacobayu — APPROVED** [review]
lgtm

**Pull request merged**